### PR TITLE
ADD SoundSamplePlayer rate

### PR DESCRIPTION
**Pull request by ninovanhooff**
Needed this so I added this to my own fork.

Tested to work.

Created this PR to check whether I didn't miss anything

### CODE REVIEW

File: src/playdate/bindings/sound.nim [hunk 1]
```diff
@@ -25,7 +25,7 @@ type PlaydateSoundFileplayer {.importc: "const struct playdate_sound_fileplayer"
     getLength {.importc: "getLength".}: proc (player: FilePlayerPtr): cfloat {.cdecl, raises: [].}
     setOffset {.importc: "setOffset".}: proc (player: FilePlayerPtr; offset: cfloat) {.
         cdecl, raises: [].}
-    # setRate* {.importc: "setRate".}: proc (player: ptr FilePlayer; rate: cfloat) {.cdecl.}
+    # setRate {.importc: "setRate".}: proc (player: FilePlayerPtr; rate: cfloat) {.cdecl.}
```
Comment (ninovanhooff) [inline src/playdate/bindings/sound.nim:28]: This change obviously doesn't do anything, but it probably points (pun intended) in the right direction

**Comment by samdze** [comment]
Looks good. Only one thing:
we should choose whether to use the `setRate` and `getRate` methods convention or the property-style getter and setter.
I think in this case the getter and setter approach (`rate` and `rate=`) is more fitting.

**Comment by ninovanhooff** [comment]
Agree the property style is best. Was wondering why you did both for some
other methods?

**Comment by samdze** [comment]
Mhh, looking at `volume` for example, I remember adding `setVolume` too because it's a setter that has two parameters.
There may be others that I added by mistake, not sure.

**Comment by ninovanhooff** [comment]
Allright, removed the getter and setter, kept property style accessors

**Comment by samdze** [comment]
Thank you!

**Pull request merged**